CMakeLists.txt

```diff
@@ -89,9 +89,9 @@ endif()
 #
 # Versions MUST contain three parts and start with lowercase 'v'.
 # Example 'v1.0.0'. They MUST not contain a pre-release label such as '-beta'.
-set(FIRMWARE_VERSION "v9.6.0")
-set(FIRMWARE_BTC_ONLY_VERSION "v9.6.0")
-set(FIRMWARE_BITBOXBASE_VERSION "v9.6.0")
+set(FIRMWARE_VERSION "v9.6.1")
```

Comment:
Didn't we say semver here doesn't really make sense? Why not v9.7.0. But if you insist, showing error codes would be a new feature anyway.

Reply:
> Didn't we say semver here doesn't really make sense?

I don't recall that. We had planned to replace this and make semvers for subsystems, but we don't have that, so this is a semver for the whole firmware API. I bumped  the patch version because no API changed. If in a future PR the API changes, the minor would be bumped.

Reply:
To elaborate, semver is strictly about API, not about feature-set.

https://webcache.googleusercontent.com/search?q=cache:2-JHjIsz8NQJ:https://semver.org/+&cd=1&hl=en&ct=clnk&gl=ch

(cache since https://semver.org/ is down for me right now)

Reply:
Oh, it's only about protobuf APIs. I wish it were documented somewhere, in the readme for example.

Reply:
Yeah, I got that. Communication. So, changing public APIs like `keystore_unlock` and extending its error enum is not covered.

Reply:
I see, didn't even consider it. But yeah I think it makes sense to treat the communication as the public API, not the rust and C libs that we use to make it.